components/infobox/wikis/starcraft2/infobox_company_custom.lua

```diff
@@ -0,0 +1,23 @@
+local Company = require('Module:Infobox/Company')
+
+local CustomCompany = {}
+
+function CustomCompany.run(frame)
+	local company = Company(frame)
+	Company.addCustomCells = CustomCompany.addCustomCells
+	return company:createInfobox(frame)
+end
+
+function CustomCompany.addCustomCells(company, infobox, args)
+	infobox:cell('Organizer Type', args.organizertype)
+	infobox:cell('Industry', args.industry)
+	infobox:cell('Partners', args.partners)
+	infobox:cell('Members', args.members)
+	infobox:cell('Key People', args['key people'])
+	infobox:cell('Products', args.products)
+	infobox:cell('Events', args.events)
+	infobox:cell('Founder', args.founder)
+	return infobox
```

Comment:
These were all intentionally left out of this infobox, so I'm not convinced about adding them back

Reply:
ah didn't know that

i just copied it from the old template on sc2

i personally think founder has some merit, happy to remove the other ones^^

Reply:
removed everything except founder
if you think founder should go too just close the pr and let me know i should kill the custom module on sc2 (as that is the only change from the base module)

Reply:
The idea is basically that that's the kind of thing you should have in your org tables, or on the esports wiki, not in an infobox